### ISSUE TITLE
[BITV] 9.2.4.1/3.1 - There are multiple "nav" landmarks (navigation areas) that lack labeling. Unique labels could be assigned using "aria-label" or "aria-labelledby" attributes. (9)

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#41865]
https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-a691d50b11-v3-n1

9.2.4.1/4.1 - https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-a691d50b11-v4-n1

9.2.4.1/5.1 - https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-a691d50b11-v5-n1

9.2.4.1/6.1 - https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-a691d50b11-v6-n1

9.2.4.1/7.1 - https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-a691d50b11-v7-n1

9.2.4.1/8.1 - https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-a691d50b11-v8-n1

9.2.4.1/9.1 - https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-a691d50b11-v9-n1

9.2.4.1/10.1 - https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-a691d50b11-v10-n1

9.2.4.1/11.1 - https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-a691d50b11-v11-n1

**Comment by Pytal** [comment]
- [ ] Add breadcrumbs label after dependency update
